package.json

```diff
@@ -3,7 +3,7 @@
   "version": "1.22.19",
   "private": true,
   "scripts": {
-    "dev": "tinacms dev --watchFolders='components/util/' -c \"next dev\"",
+    "dev": "tinacms dev -c \"next dev\"",
```

Comment:
why get rid of the watch?

Reply:
I didn't see the doc mentioning this argument anywhere, and it's already watching the whole project by default. Is there any particular reason that we're setting this arg and only watching the util folder?

Reply:
ok cool